orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -197,11 +195,7 @@ def _get_session_config(self):
         # default to PRIORITY_BASED execution order
         session_options.execution_order = onnxruntime.ExecutionOrder.PRIORITY_BASED
         # 0:Verbose, 1:Info, 2:Warning. 3:Error, 4:Fatal. Default is 2.
-        session_options.log_severity_level = int(self._loglevel)
-
-        # enable dumping optimized training graph
-        if self._save_onnx:
-            session_options.optimized_model_filepath = self._save_onnx_prefix + '_training_optimized.onnx'
```

Comment:
@SherlockNoMad, please let me know if this model adds any value to be saved. We are currently saving:
- The exported model.
- The optimized model got after graph transformations as defined in ```ortmodule_graph_builder```.

Reply:
Please keep it. There can be bugs in the graph transformers that this model would reveal.
This and other ONNX models were requested by our engineers during their debug on model shipping tasks

Reply:
I think you might be right here. We are using "training" word in GraphExecutionManager, which is wrong.
Check whether this is done on both inference/training manager already - if it does, this is redundant. If not, we can add this to training manager as opposed to here

We must have torch_exported and inference_optimized in inference manager
We must have training and training_optimized in training manager

Reply:
Excluding this onnx model, there are 4 models that are saved:
- torch_exported_training.onnx
- optimized_training.onnx
- torch_exported_inference.onnx
- optimized_inference.onnx

I think this model here is very similar to the ```optimized_training.onnx``` but it might not be the same.

I can save this model for both training and inference. That would make 3 models on training and 3 on inference side. My only concern is that this model is very similar to the optimized version that I mentioned earlier.

Reply:
Let's remove this onnx model for now. If we think we need it later, we can add it back.